components/src/utils.ts

```diff
@@ -75,3 +80,15 @@ export const getWebMonetizationLinkHref = (userAgent: string): string => {
   }
   return 'https://webmonetization.org/'
 }
+
+export function isInteractionSuccess(
+  params: PaymentStatus
+): params is PaymentStatusSuccess {
+  return 'interact_ref' in params
+}
+
+export function isInteractionRejected(
+  params: PaymentStatus
+): params is PaymentStatusRejected {
+  return 'result' in params && params.result === 'grant_rejected'
+}
```

Comment:
If these are used only in widget/views/interaction.ts, let's keep them there only.